### ISSUE TITLE
Improve contrast accessibility

### ISSUE THREAD

**Issue by brandstetterm** [inovex/scrumlr.io#2852]

**Comment by Resaki1** [comment]
additional idea: add a "high contrast" mode in addition to light and dark mode, which further increases contrast (and maybe also makes transparent backgrounds full-color)

**Comment by brandstetterm** [comment]
> additional idea: add a "high contrast" mode in addition to light and dark mode, which further increases contrast (and maybe also makes transparent backgrounds full-color)

Good idea 👍🏼 I've already thought about the feature that you can chose from multiple themes. A high contrast theme could be one of them.

**Comment by SelinaBuff** [comment]
**Description:** The goal of this issue is to improve accessibility regarding contrast in the application.

**1) Review the Board View _(see Screenshot 1)_:**
- Check the contrast ratios in the board view (see Screenshot 1).
- Ensure that all text, buttons, and interactive elements meet the recommended contrast ratios for accessibility (at least 4.5:1 for normal text and 3:1 for large text or interactive elements).

2) **Review the Settings Dialog** _(see Screenshot 2)_:
- Check the contrast ratios in the settings dialog
- Make sure the contrast ratios are optimized according to accessibility guidelines here as well.

**Acceptance Criteria:**
- All text and interactive elements are readable and accessible, both in the board view and the settings dialog.

**Additional Notes:**
- Please document findings and improvements in an organized manner using platforms such as Miro or Figma for clarity.



![Image](https://github.com/user-attachments/assets/a6ebe2fa-c499-4005-b074-edc0c0043de1)



![Image](https://github.com/user-attachments/assets/64cfb543-bc6a-4bf9-8b1f-e1615133337f)
